pkgs/development/compilers/gcc/4.9/default.nix

```diff
@@ -329,6 +329,7 @@ stdenv.mkDerivation ({
     maintainers = with lib.maintainers; [ veprbl ];
 
     platforms = lib.platforms.unix;
+    badPlatforms = [ "aarch64-darwin" ];
```

Comment:
```suggestion
    broken = [ "aarch64-darwin" ];
```

Reply:
What's the difference between broken and unsupported (i.e. badPlatforms)?

Reply:
@veprbl yes, meta.broken is correct here because badPlatforms is an more or less undocumented field to disable builds on hydra and not for end users. With this PR merged hydra is not building gcc49 for m1 macs anymore but end users can happily try it and waste their time.

The syntax is incorrect though.

Reply:
I can fix the syntax if it's incorrect. I'm not sure what it should be – I copied it from other places and it works on my M1 machine.

Reply:
`badPlatforms` is perfectly fine here. `broken` in its meaning is more like "it should work (intended to), but, for some reason, it doesn't", which is not the case here